python/pyarrow/tests/test_schema.py

```diff
@@ -87,3 +87,10 @@ def test_schema_equals(self):
         del fields[-1]
         sch3 = A.schema(fields)
         assert not sch1.equals(sch3)
+
+
+class TestField(unittest.TestCase):
+    def test_empty_field(self):
+        f = arrow.Field()
+        # This used to segfault on empty field
+        repr(f)
```

Comment:
`Field` is constructed in two sages (schema.pyx:131):
{code}
        cdef Field result = Field()
        result.init(self.schema.field(i))
{code}

And there's also `Field.from_py` which also creates an empty field.

How can I tell between these cases to ones where this is not OK. Maybe remove the `Field` import from pyarrow `__init_.py`?